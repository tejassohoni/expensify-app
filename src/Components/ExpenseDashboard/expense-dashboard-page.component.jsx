import React from "react";

import ExpensesSummary from "../ExpensesSummary/expenses-summary.component";
import ExpenseListFilters from "../ExpenseListFilters/expense-list-filters.component";
import ExpenseList from "../ExpenseList/expense-list.component";

const ExpenseDashboard = () => (
  <div>
    <ExpensesSummary />
    <ExpenseListFilters />
    <ExpenseList />
  </div>
);

export default ExpenseDashboard;
