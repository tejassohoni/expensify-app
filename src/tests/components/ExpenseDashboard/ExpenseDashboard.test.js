import React from "react";
import { shallow } from "enzyme";
import ExpenseDashboard from "../../../Components/ExpenseDashboard/expense-dashboard-page.component.jsx";

test("Should render ExpenseListItem", () => {
  const wrapper = shallow(<ExpenseDashboard />);
  expect(wrapper).toMatchSnapshot();
});
