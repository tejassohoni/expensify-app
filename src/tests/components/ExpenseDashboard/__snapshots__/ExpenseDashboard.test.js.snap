// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem 1`] = `
<div>
  <React.Memo />
  <React.Memo />
  <React.Memo />
</div>
`;
